rust/README.md

```diff
@@ -65,3 +65,5 @@ and check for lint issues:
 ```bash
 cargo +stable fmt --all -- --check
 ```
+
+hi
```

Comment:
```suggestion
```

Reply:
This was just a test, I should have dropped this commit.